fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/impl/test/ResourceReferenceDAOTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.impl.test;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.logging.Logger;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import com.ibm.fhir.database.utils.derby.DerbyMaster;
+import com.ibm.fhir.database.utils.derby.DerbyTranslator;
+import com.ibm.fhir.database.utils.model.DbType;
+import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
+import com.ibm.fhir.persistence.jdbc.FHIRPersistenceJDBCCache;
+import com.ibm.fhir.persistence.jdbc.cache.CommonTokenValuesCacheImpl;
+import com.ibm.fhir.persistence.jdbc.cache.FHIRPersistenceJDBCCacheImpl;
+import com.ibm.fhir.persistence.jdbc.cache.NameIdCache;
+import com.ibm.fhir.persistence.jdbc.connection.FHIRDbFlavor;
+import com.ibm.fhir.persistence.jdbc.connection.FHIRDbFlavorImpl;
+import com.ibm.fhir.persistence.jdbc.dao.impl.ResourceReferenceDAO;
+import com.ibm.fhir.persistence.jdbc.postgresql.DerbyResourceReferenceDAO;
+import com.ibm.fhir.schema.derby.DerbyFhirDatabase;
+
+/**
+ * Unit tests for {@link ResourceReferenceDAO}
+ */
+public class ResourceReferenceDAOTest {
```

Comment:
Where are the unit tests? is this suppose to be abstract?

Reply:
They were no longer relevant after one of my refactors. There's plenty of coverage from existing tests so I'm going to remove.